v2.0/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -2,229 +2,286 @@
+   to each app container.
+-  The global policy resource "databases" uses the selector to select which
+   containers to apply the policy to - in this case those that have the 
+   `role = database` label.  The ingress and egress policy applied to these
+   containers is as follows:
+   -  An ingress rule to allow TCP traffic to port 3306 from endpoints that have
+      the label `role = app` (i.e. from app containers since they are the only
+      ones with the `role = app` assignment)
+   -  An ingress and egress rule to allow all traffic from and to endpoints that
+      have the label `role = database` (i.e. from database containers).
+-  The global policy resource "apps" uses the selector to select which
+   containers to apply the policy to - in this case those that have the 
+   `role = app` label.  The policy applied to these containers is a single rule 
+   to allow all egress traffic.
 
-For more details about advanced policy options read the 
-[Advanced Network Policy tutorial]({{site.baseurl}}/{{page.version}}/usage/configuration/advanced-network-policy).
+## Multiple networks
 
+Whilst the Docker API supports the ability to attach a container to multiple
+networks, it is not possible to use this feature of Docker when using the Calico.
+
+However, using the selector-based approach for defining network policy it is
+possible to achieve the same effect of overlapping networks but with a far 
+richer policy set.
+
+Extending the previous example, suppose we introduce another network that is
+used for system backups and that we want some of our database containers to be
+on both the database network and the backup network (so that they are able to 
+back up the database).  
+
+One approach for doing this is as follows:
+
+-  Define a new label, say `backup = true` to indicate that a particular
+   endpoint should be allowed access to the backup network.
+-  Define global policy "backupnetwork" that allows full access between all 
+   components with the  `backup = true` label.
+-  Create a Docker network "backups" for backups and update the associated 
+   profile to assign the `backup = true` label
+-  Create a Docker network "database-backup" for databases _and_ backup access,
+   and update the associated profile to assign both the `backup = true` and 
+   `role = database` labels.
+   
+For your database containers that also need to be on the backup network, use the 
+"database-backup" network.  Since containers in this network will have the two
+labels assigned to it, they will pick up policy that selects both labels - in
+other words they will have the locked down database access plus access to the
+backup network.
+
+Obviously, the example of allowing full access between everything on the "backup"
+network is probably a little too permissive, so you can lock down the access within
+the backup network by modifying the global policy selected by the `backup = true`
+label.
```

Comment:
I think multiple label is worth discussing especially WRT to overlapping networks.... but I wonder if I should beef this up with more sample YAML, or leave as is.  Was worried about this getting too long.   I suppose I could split into two docs profile based and selector based....   WDYT?